src/test/resources/json/WikiTest/testWikiMapping/files/57d87dd6da3e2401ed91bbfe/versions/index.json

```diff
@@ -0,0 +1,26 @@
+{
+    "data": [
+        {
+            "links": {
+                "self": "http://localhost:8000/v2/files/57d87dd6da3e2401ed91bbfe/versions/1/",
+                "html": "https://test.osf.io/ng9em/files/osfstorage/57d87dd6da3e2401ed91bbfe?revision=1"
```

Comment:
I'd just go ahead and globally switch all instances of 'test.osf.io' to 'localhost:8000', even if we aren't using the 'html' relation right now.  It could be in the future that we _would_ use it, and it would be nice to be consistent with the URLs presented in the test JSON, so they can be easy to reason about if we were to use them in the future.